tests/unit/grpc_internals/zeebe_process_adapter_test.py

```diff
@@ -154,3 +154,34 @@ async def test_calls_open_in_rb_mode(self, zeebe_adapter: ZeebeProcessAdapter):
         await zeebe_adapter.deploy_process(file_path)
 
         self.open_mock.assert_called_with(file_path, "rb")
+
+
+@pytest.mark.asyncio
+class TestDeployResource:
+    open_mock: MagicMock
+
+    @pytest.fixture(autouse=True)
+    def mocked_aiofiles_open(self):
+        read_mock = AsyncMock(return_value=bytes())
+
+        file_mock = AsyncMock()
+        file_mock.__aenter__.return_value.read = read_mock
+
+        with patch("pyzeebe.grpc_internals.zeebe_process_adapter.aiofiles.open", return_value=file_mock) as open_mock:
+            self.open_mock = open_mock
+            yield
```

Comment:
I suggest we make this a common fixture not scoped to a single class to avoid duplication with `TestDeployProcess`

Reply:
Done